common/src/main/java/org/wso2/testgrid/common/util/S3StorageUtil.java

```diff
@@ -50,13 +65,54 @@ public static String getS3BucketURL() {
         return String.join("/", TestGridConstants.AMAZON_S3_URL, s3BucketName);
     }
 
+    public static void deleteTestPlan(TestPlan testPlan) {
+
+        try {
+            ArtifactReadable artifactDownloadable = new AWSArtifactReader(ConfigurationContext.
+                    getProperty(ConfigurationContext.ConfigurationProperties.AWS_REGION_NAME),
+                    ConfigurationContext.getProperty(ConfigurationContext.ConfigurationProperties.AWS_S3_BUCKET_NAME));
+
+            String s3KeyName = deriveS3TestPlanDirPath(testPlan, artifactDownloadable);
+            logger.info("Files cleaned in :" + s3KeyName);
+            String clientRegion = ConfigurationContext.getProperty
+                    (ConfigurationContext.ConfigurationProperties.AWS_REGION_NAME);
+            String bucketName = ConfigurationContext.getProperty
+                    (ConfigurationContext.ConfigurationProperties.AWS_S3_BUCKET_NAME);
+
+            AmazonS3 s3Client = AmazonS3ClientBuilder.standard()
+                    .withCredentials(new ProfileCredentialsProvider())
```

Comment:
We have been using the AWS SDK's PropertiesFileCredentialsProvider for credentials. Why this change?

https://github.com/wso2/testgrid/blob/46f29344e230807f556eefd356fc8f5c030e4267/infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/aws/AMIMapper.java#L71